receipt_dynamo/example_migration_receipt.py

```
@@ -0,0 +1,134 @@
+    - TransactionalOperationsMixin: update_receipts transactionally
+    - QueryByTypeMixin: list_receipts using GSITYPE
+    - CommonValidationMixin: _validate_image_id, _validate_receipt_id
+
+    All existing methods remain available and work identically.
+    """
+
+    # All your existing methods work exactly the same
+    # No changes needed to the implementation
+
+    @handle_dynamodb_errors("get_receipt")
+    def get_receipt(self, image_id: str, receipt_id: int) -> Receipt:
```

Ruff (0.12.2)

Undefined name `Receipt`

(F821)